docs/standards/news.rst

```diff
@@ -3,22 +3,119 @@
 Schema Release Notes
 ================================================================================
 
-Version 1.4.0-dev: (In development)
+Version 1.4.0:
 --------------------------------------------------------------------------------
 
-**Version 1.4-dev, In development.**
-
-New Schema:
-
-1. Introduced the ``RepertoireGroup`` Schema for describing sets of repertoires
-   to be analyzed together. Has support for defining a time series using ``TimePoint``.
+**Version 1.4 schema release.**
+
+New General Purpose Schema:
+
+1. Introduced the experimental ``DataFile`` object, which defines a JSON file
+   holding Repertoire metadata, data processing analysis objects, or any object
+   in the AIRR Data Model.
+2. Introduced the experimental ``RepertoireGroup`` Schema for describing sets
+   of repertoires to be analyzed together.
+3. Introduced the ``InfoObject`` object, which provides information about
+   data and ADC API responses.
+4. Introduced the ``TimePoint`` Schema for defining the time point at which an
```

Comment:
Is this already in wider productive use? Or should it still be flagged as experimental for now?

Reply:
it's experimental and was added in conjunction with RepertoireSet (formerly RepertoireGroup) to define a group of repertoires in a time series.

Reply:
Are we talking about `TimePoint` or `InfoObject`?

Reply:
`TimePoint`

Reply:
Flagged as experimental in 50a9d9b.